src/components/tools/Charts/DotAndWhisker/Chart.svelte

```diff
@@ -2,12 +2,16 @@
   import { SkeletonText, SkeletonPlaceholder } from "carbon-components-svelte";
   import { LayerCake, Svg, Html } from "layercake";
   import { scaleBand } from "d3-scale";
-  import { timeFormat, timeParse } from "d3-time-format";
   import { min, max, groups } from "d3-array";
   import { setContext } from "svelte";
   import { writable } from "svelte/store";
+  import { isEmptyData } from "~/helpers/utilities";
```

Comment:
you could remove this unused import

Reply:
removed in https://github.com/berkeley-gif/caladapt-website-2021/pull/211/commits/25d6598edcd36bedcc3ee572787c9f1eb621d9dd